### ISSUE TITLE
Please update PSGallery with 1.3.0

### ISSUE THREAD

**Issue by scwat-github** [EliteLoser/WriteAscii#8]
Hi

Any chance you could push the Sort-Object fix to PSGallery please? 1.2.2.1 is there and it's still showing as Sort on one of the lines, bit painful fixing it in some ADO pipelines. Thanks for the module!

```
$LetterArray = [string[]]($Letters.GetEnumerator() | Sort Name | Select -ExpandProperty Name)
```

**Comment by EliteLoser** [comment]
Yeah, I will fix it one of these days. Didn't realize it was mission-critical for any enterprise! :D Need to boot into Windows...

**Comment by scwat-github** [comment]
It's great for separating sections out in the logs :)

**Comment by EliteLoser** [comment]
Status update: Today I tried to boot into Windows. Long story short: after 5 bluescreens (some of which were displayed distorted) and a broken install trying to update, I failed to access Windows in the end. I think I need a clean install. Don't have a Windows license - and my company is no longer income, but an expense to keep the server running, so it could take a while...

**Comment by EliteLoser** [comment]
As for fixing it after download, it's not too hard (although atrocious stylistically - I tried to gain access to Windows so I could fix it, but failed...).

I tested a simple, exact-match regex on my command line:

```
'$LetterArray = [string[]]($Letters.GetEnumerator() | Sort Name | Select -ExpandProperty Name)' -replace '\$LetterArray = \[string\[\]\]\(\$Letters\.GetEnumerator\(\) \| Sort Name \| Select -ExpandProperty Name\)', '$LetterArray = [string[]]($Letters.GetEnumerator() | Sort-Object -Property Name | Select-Object -ExpandProperty Name)'

$LetterArray = [string[]]($Letters.GetEnumerator() | Sort-Object -Property Name | Select-Object -ExpandProperty Name)
```

So you could use that on the whole file, to make the code simpler (after all the line isn't elsewhere and therefore doesn't match elsewhere, and if it did, I guess you would want this anyway).

Such as:

```
(Get-Content -LiteralPath $OldFile) -replace '\$LetterArray = \[string\[\]\]\(\$Letters\.GetEnumerator\(\) \| Sort Name \| Select -ExpandProperty Name\)', '$LetterArray = [string[]]($Letters.GetEnumerator() | Sort-Object -Property Name | Select-Object -ExpandProperty Name)' | Set-Content -Encoding utf8 -Path $NewFixedFile
```

Yeah, I wish I could fix it though...

**Comment by turbosnute** [comment]
My workaround was to make "sort" an alias for "Sort-Object" in the Powershell Profile ($PROFILE)
```powershell
Set-Alias "sort" "Sort-Object"
```

**Comment by EliteLoser** [comment]
Attention, please! I have an announcement to make! *fanfare plays loudly in the background* Today, 2026-04-26 AD, the WriteAscii module was updated in the PowerShell gallery. You no longer need workarounds. Sorry about the procrastination.

From a session on a computer where I didn't have the module previously, installed after publishing the update:

```
PS C:\> Install-Module -Name WriteAscii -Scope CurrentUser

PS C:\> Write-Ascii test
 _               _
| |_   ___  ___ | |_
| __| / _ \/ __|| __|
| |_ |  __/\__ \| |_
 \__| \___||___/ \__|

PS C:\> (Get-Module -Name WriteAscii).Version

Major  Minor  Build  Revision
-----  -----  -----  --------
1      3      0      -1      



PS C:\> (Get-Module -Name WriteAscii).Version.ToString()
1.3.0

```

Yes! At long last! Can you even process news this good?! :D This calls for a minor celebration. Everyone is encouraged to pop a bottle of slightly fine red wine, and to thoroughly enjoy it while playing with this awesome ASCII art module.

From here on, the future looks bright for us all due to this!
